spyder_remote_services/services/envs_manager/handlers.py

```diff
@@ -0,0 +1,145 @@
+from jupyter_server.base.handlers import JupyterHandler
```

Comment:
```suggestion
from jupyter_server.auth.decorator import authorized
from jupyter_server.base.handlers import JupyterHandler
```

Move import to the right place.